Makefile

```diff
@@ -268,10 +353,9 @@ plugins:
 #
 # Usage:
 #	make post-checkout-hook [DOCKERFILE=]
-
 post-checkout-hook:
 	if [ -n "$(findstring /arm64/,$(DOCKERFILE))" ]; then \
-		mkdir -p $(DOCKERFILE)/hooks; \
+		mkdir -p docker-image/$(DOCKERFILE)/hooks; \
```

Comment:
Good catch!
I wondered why broken directories are created after `make src-all` for a while ago. This is the answer.